hangups/http_utils.py

```diff
@@ -46,6 +46,9 @@ def fetch(method, url, params=None, headers=None, cookies=None, data=None,
         else:
             error_msg = None
             break
+        finally:
+            if 'res' in locals():
+                res.close()
```

Comment:
> The session used in aiohttp.request is attached to the aiohttp.ClientResponse but not closed by anyone - only a direct access to the protected member aiohttp.ClientResponse._session.close() can close it.

So in aiohttp 2.2.0 it's not possible to avoid the resource warning from `aiohttp.request` at all?